package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "whirli-client",
-  "version": "0.0.27",
+  "version": "0.0.28",
```

Comment:
Looks like this is version 0.0.28 and the CHANGELOG didn't get updated previously.

Could you edit the CHANGELOG and this PR title to reflect that please

Reply:
Yeah, I was a bit confused about that. Will change.

Reply:
all sorted